server/src/routes/course/crossCheck/isSubmissionDeadlinePassed.ts

```diff
@@ -0,0 +1,5 @@
+export const isSubmissionDeadlinePassed = (studentEndDate: string) => {
```

Comment:
probably it is better to just put this function into tasks.service instead of creating new file per function

Reply:
in htis case it would be better to pass not date but task itself

Reply:
Done